modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/info/item/provider/CommerceOrderInfoItemFieldValuesProvider.java

```diff
@@ -579,6 +585,10 @@ private List<InfoFieldValue<Object>> _getCommerceOrderInfoFieldValues(
 				new InfoFieldValue<>(
 					CommerceOrderInfoItemFields.totalDiscountAmountInfoField,
 					commerceOrder.getTotalDiscountAmount()));
+			commerceOrderInfoFieldValues.add(
+				new InfoFieldValue<>(
+					CommerceOrderInfoItemFields.totalWithTaxAmountInfoField,
+					commerceOrder.getTotalWithTaxAmount()));
```

Comment:
@victorKammerer this statement should came after the one related to `commerceOrder.getTotalDiscountWithTaxAmount()` to keep the asc sorting